stratagem-runner/src/stratagem_purge.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+// Usage: stratagem_purge FS [NID1] [NID2] ...
+
+use liblustreapi;
+use std::env;
+use std::process;
+
+fn main() {
+    let mut args = env::args();
+    let device = args.nth(1).expect("No device specified");
+
+    let fidlist: Vec<_> = args.map(|arg| arg).collect();
+    if let Err(e) = liblustreapi::rmfid(&device, &fidlist[..]) {
```

Comment:
With the change to rmfid, this can just be:

```suggestion
    if let Err(e) = liblustreapi::rmfid(&device, args) {
```